src/ripple/shamap/impl/SHAMapSync.cpp

```diff
@@ -419,7 +419,7 @@ SHAMapAddNode SHAMap::addRootNode (Blob const& rootNode,
 
     assert (seq_ >= 1);
     auto node = SHAMapAbstractNode::make(
-        rootNode, 0, format, uZero, false, f_.journal ());
+        rootNode, 0, format, SHAMapHash{uZero}, false, f_.journal ());
```

Comment:
If we had `uint256` up to speed on `constexpr` already, I would do this.  But as is, I think the compiler-generated flag on the `static` would actually make things worse.